user-guide/live-traffic/inspector-types.md

```diff
@@ -31,53 +39,49 @@ Every HTTP request begins with plaintext headers that describe what the client r
+Like the HTTP request, every HTTP response begins with plaintext headers that describe the result from the request. The first line of the response (the _"Status line"_) contains the HTTP version (for example __"HTTP/1.1"__), the response status code (for example __"200"__), and the response status text (for instance __"OK"__). Following the _"Status line"_ there are one or more lines containing name-value pairs of metadata about the response and the server, such as the length of the response file, the Content-Type, and how the response may be cached.
 
 
 ## Text Inspector
 
-The __Text__ inspector allows you to view the request and response bodies as text. The __Text__ Inspector truncates the data it shows at the first null byte it finds, making it an inappropriate choice for displaying binary content.
-
+The **Text inspector** allows you to view the request and response bodies as text. The **Text Inspector **truncates the data it shows at the first null byte it finds, making it an inappropriate choice for displaying binary content.
 Most of the Inspector is a large text area that displays the body text interpreted using the character set detected using the headers, the byte-order-marker, or an embedded META tag declaration.
 
 ![Text Inspector](../../images/livetraffic/inspectors/inspectors-textview.png)
 
 ## Image Inspector
 
-The __Image__ Inspector (__Response only__) allows you to view the response bodies as an image. The Inspector can display most common web image formats, including JPEG, PNG, and GIF, as well as less common formats like cursors, WebP, JPEG-XR, bitmaps, TIFF. The Inspector does not support displaying SVG graphics (SVG responses can be viewed using the __Web__ inspector).
+The **Image Inspector** (_Response_ only) allows you to view the response bodies as an image. The Inspector can display most common web image formats, including JPEG, PNG, GIF, and less common formats like cursors, WebP, JPEG-XR, bitmaps, TIFF. The Inspector does not support displaying SVG graphics (SVG responses can be viewed using the **Web inspector**).
 
 ![Image Inspector](../../images/livetraffic/inspectors/inspectors-image.png)
 
 ## WebForms Inspector
 
-The __WebForms__ inspector (__Request only__) parses the request's query string and body for any HTML form-data. If a form is found, it is parsed, and the name/value pairs are displayed in the grid view. The Inspector works best with _application/x-www-form-urlencoded_ data used by most simple web forms.
+The **WebForms inspector** (_Request_ only) parses the request's query string and body for any HTML form-data. If a form is found, it is parsed, and the name/value pairs are displayed in the grid view. The Inspector works best with application/x-www-form-urlencoded data used by most simple web forms.
```

Comment:
Additionally, with this release we've added the ability to copy data from the tables in this inspector, maybe it is a good idea to mention it here as it is available only through context menu. The feature is described in [this JIRA item](https://kinvey.atlassian.net/browse/FID-1501) - check the Acceptance Criteria for all the details of the implementation.
Also, here's a video demonstrating the feature: https://progresssoftware-my.sharepoint.com/:v:/g/personal/vladimirov_progress_com/ET-yEQNV8vBOtmF98gdngMMBJNBUAzFEVhUL5I6KOd-b_Q?e=ygwdnY